.github/workflows/tpu-tests.yml

```diff
@@ -25,8 +26,11 @@ env:
 jobs:
   test-on-tpus:
     runs-on: ubuntu-22.04
-    # run only when the PR title contains '[TPU]' or is a merge to master
-    if: ${{ startsWith(github.event_name, 'pull_request') && contains(github.event.pull_request.title, '[TPU]') || (github.event_name == 'push' && github.ref == 'refs/heads/master') }}
+    # run only when the PR title contains 'TPU' or is a merge to master
+    if: |
+      (github.event_name == 'push' && github.ref == 'refs/heads/master') ||
```

Comment:
@Borda This seems to have broken the job. I don't see it running on master commits. Can you fix this please?